api/tools/src/main/java/org/pmiops/workbench/tools/ApiClientFactory.java

```diff
@@ -0,0 +1,34 @@
+package org.pmiops.workbench.tools;
+
+import java.io.IOException;
+import org.pmiops.workbench.firecloud.ApiClient;
+import org.pmiops.workbench.firecloud.api.*;
+
+public abstract class ApiClientFactory {
+
+  protected ApiClient apiClient;
```

Comment:
Using a protected constructor seems preferable to setting protected properties like this, in my view